BoardConfig.mk

```diff
@@ -129,6 +129,9 @@ TARGET_USE_CUSTOM_LUN_FILE_PATH             := /sys/class/android_usb/android0/f
 BOARD_HAS_NO_SELECT_BUTTON                  := true
 BOARD_HAS_LARGE_FILESYSTEM                  := true
 TARGET_USERIMAGES_USE_EXT4                  := true
+ifeq ($(BUILD_TWRP), true)
+TARGET_USERIMAGES_USE_F2FS                  := true
+endif
```

Comment:
If CM11 don't have F2FS then just simply put it there, it will not have any effect